### PR TITLE
Select video bar changes

### PR DESCRIPTION
**Pull request by mr-mr**
A tweak to the select video bar when choosing a video.
Making it stand out a bit more, and pulling in a couple of elements to ensure ed's are picking the correct video quickly. Thanks @clloyd for the react input!

image

No image example
image

cc @akash1810 @ShaunYearStrong @Reettaphant

### CODE REVIEW

File: public/video-ui/styles/components/_bar.scss [hunk 1]
```diff
@@ -21,14 +21,31 @@
...
+      margin-right: 15px;
+
+  }
+
+  &__image-placeholder {
+      background-color: $color800Grey;
+      width: 160px;
+      height: 90px;
+      text-align: center;
+      padding-top: 32px;
```
Comment (akash1810) [inline public/video-ui/styles/components/_bar.scss]: Any reason for 32? Its a bit of an odd number...

**Comment by mr-mr** [comment]
This change was to reflect the fact that bars have different purposes, and should separate either by distance or visually. I agree on the vertical height, I'll attempt to make it thinner.

File: public/video-ui/styles/components/_bar.scss (hunk 1 above)
Comment (mr-mr) [inline public/video-ui/styles/components/_bar.scss]: It's to have the 'No Image' title in the centre of the blank image box. I'll upload an example screenshot

**Comment by paperboyo** [comment]
Good yellow! Isn't the font in buttons and search box wrong (Helvetica instead of Agate), though?

**Comment by mr-mr** [comment]
Small steps @paperboyo small steps

**Comment by mr-mr** [comment]
@akash1810 
Thinner up top now. Can always improve!
image

**Pull request merged**